contrib/platform/src/com/sun/jna/platform/win32/Shell32Util.java

```diff
@@ -112,4 +115,28 @@ public static final String getSpecialFolderPath(final int csidl, final boolean c
             throw new Win32Exception(Kernel32.INSTANCE.GetLastError());
         return Native.toString(pszPath);
     }
+
+    /**
+     * Parses a command line string and returns an array of Strings of the command
+     * line arguments.
+     *
+     * @param cmdLine
+     *            A string that contains the full command line. If this parameter is
+     *            an empty string the function returns the path to the current
+     *            executable file.
+     * @return An array of {@link LPWSTR} values, similar to {@code argv}.
+     */
+    public static final String[] CommandLineToArgv(String cmdLine) {
+        WString cl = new WString(cmdLine);
+        IntByReference nargs = new IntByReference();
+        LPWSTR.ByReference strArr = Shell32.INSTANCE.CommandLineToArgvW(cl, nargs);
```

Comment:
See `Shell32.java` for comment

```suggestion
        Pointer strArr = Shell32.INSTANCE.CommandLineToArgvW(cl, nargs);
```